integration/system/disk_usage_test.go

```diff
@@ -53,20 +54,18 @@ func TestDiskUsage(t *testing.T) {
 				assert.NilError(t, err)
 				assert.Assert(t, du.LayersSize > 0)
 				assert.Equal(t, len(du.Images), 1)
-				assert.DeepEqual(t, du, types.DiskUsage{
-					LayersSize: du.LayersSize,
-					Images: []*types.ImageSummary{
-						{
-							Created:  du.Images[0].Created,
-							ID:       du.Images[0].ID,
-							RepoTags: []string{"busybox:latest"},
-							Size:     du.LayersSize,
-						},
-					},
-					Containers: []*types.Container{},
-					Volumes:    []*volume.Volume{},
-					BuildCache: []*types.BuildCache{},
-				})
+				assert.Equal(t, len(du.Images[0].RepoTags), 1)
+				assert.Equal(t, du.Images[0].RepoTags[0], "busybox:latest")
```

Comment:
```suggestion
				assert.Check(t, is.Equal(du.Images[0].RepoTags[0], "busybox:latest"))
```

Reply:
We tend to use `gotest.tools/v3/assert/cmp`-style asserts whenever it's possible.